enterprise/app/filter/filter_util.tsx

```diff
@@ -138,3 +144,48 @@ export function toStatusParam(statuses: Iterable<invocation.OverallStatus>): str
     .sort((a, b) => statusFromString(a) - statusFromString(b))
     .join(" ");
 }
+
+export const DURATION_SLIDER_VALUES = [
+  0,
+  1,
+  2,
+  3,
+  4,
+  5,
+  10,
+  15,
+  20,
+  25,
+  30,
+  40,
+  50,
+  60,
+  120,
+  180,
+  240,
+  300,
+  600,
+  900,
+  1200,
+  1500,
+  1800,
+  2400,
+  3000,
+  3600,
```

Comment:
(uber nit, just to make these easier to read / tweak in the future)

```suggestion
  2 * 60,
  3 * 60,
  4 * 60,
  5 * 60,
  10 * 60,
  15 * 60,
  20 * 60,
  25 * 60,
  30 * 60,
  40 * 60,
  50 * 60,
  60 * 60,
```